dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-6.0/io.openliberty.microProfile-6.0.feature

```diff
@@ -0,0 +1,26 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.microProfile-6.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-ShortName: microProfile-6.0
+Subsystem-Name: MicroProfile 6.0
+-features=\
+  io.openliberty.appSecurity-4.0, \
+  io.openliberty.cdi-3.0, \
+  io.openliberty.jsonb-2.0, \
+  io.openliberty.jsonp-2.0, \
+  io.openliberty.restfulWS-3.0, \
+  com.ibm.websphere.appserver.servlet-5.0, \
```

Comment:
Can these be changed to appSecurity-5.0, cdi-4.0, jsonb-3.0, jsonp-2.1, restfulWS-3.1 and servlet-6.0?

Reply:
well they can't if we are going to do EE9 toleration for now, but then there should be tolerates in here.